### ISSUE TITLE
https://docs.rapids.ai/deployment/nightly/guides/azure/infiniband.html

### ISSUE THREAD

**Issue by jacobtomlinson** [rapidsai/deployment#199]

**Comment by jacobtomlinson** [comment]
I'm unable to test this as I can't seem to create an `ND40rs_v2` instance. @mmccarty did you have to request this specifically somehow?

**Comment by mmccarty** [comment]
I don't believe so. Although, I've ran into that in the past and just used a different instance with GPUs
